PocoNET TestSuite fails to build with Visual Studio projects

The testsuite fails to build with

`1>SocketsTestSuite.obj : error LNK2019: unresolved external symbol "public: static class CppUnit::Test * __cdecl SocketProactorTest::suite(void)" (?suite@SocketProactorTest@@SAPEAVTest@CppUnit@@XZ) referenced in function "public: static class CppUnit::Test * __cdecl SocketsTestSuite::suite(void)" (?suite@SocketsTestSuite@@SAPEAVTest@CppUnit@@XZ)`

The offending line is: https://github.com/pocoproject/poco/blob/devel/Net/testsuite/src/SocketsTestSuite.cpp#L35

I noticed that the SocketProactor.h/.cpp files are not in the Net project and that the respective test classes are also not in the TestSuite project, but all the 4 files are in the filesystem.
I couldn't find documentation online about the SocketProactor class. So my question is:
- Is this a deprecated class and the fix is just to remove the offending line, or
- Should the class and its unit test be added to `TestSuite_vs90.vcproj` and then ProGen'd ?

--- a/Net/Net_vs170.vcxproj
+++ b/Net/Net_vs170.vcxproj
@@ -909,6 +909,7 @@
     <ClInclude Include="include\Poco\Net\SocketImpl.h"/>
     <ClInclude Include="include\Poco\Net\SocketNotification.h"/>
     <ClInclude Include="include\Poco\Net\SocketNotifier.h"/>
+    <ClInclude Include="include\Poco\Net\SocketProactor.h"/>
     <ClInclude Include="include\Poco\Net\SocketReactor.h"/>
     <ClInclude Include="include\Poco\Net\SocketStream.h"/>
     <ClInclude Include="include\Poco\Net\SSPINTLMCredentials.h"/>
@@ -1197,6 +1198,9 @@
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
     </ClCompile>
     <ClCompile Include="src\SocketNotifier.cpp">
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>
+    </ClCompile>
+    <ClCompile Include="src\SocketProactor.cpp">
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
     </ClCompile>
     <ClCompile Include="src\SocketReactor.cpp">

--- a/Net/Net_vs170.vcxproj.filters
+++ b/Net/Net_vs170.vcxproj.filters
@@ -420,6 +420,9 @@
     <ClInclude Include="include\Poco\Net\SocketReactor.h">
       <Filter>Reactor\Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="include\Poco\Net\SocketProactor.h">
+      <Filter>Reactor\Header Files</Filter>
+    </ClInclude>
     <ClInclude Include="include\Poco\Net\MailMessage.h">
       <Filter>Mail\Header Files</Filter>
     </ClInclude>
@@ -750,6 +753,9 @@
       <Filter>Reactor\Source Files</Filter>
     </ClCompile>
     <ClCompile Include="src\SocketReactor.cpp">
+      <Filter>Reactor\Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="src\SocketProactor.cpp">
       <Filter>Reactor\Source Files</Filter>
     </ClCompile>
     <ClCompile Include="src\MailMessage.cpp">

--- a/Net/Net_vs90.vcproj
+++ b/Net/Net_vs90.vcproj
@@ -1251,6 +1251,10 @@
 					RelativePath=".\include\Poco\Net\SocketReactor.h"
 					>
 				</File>
+				<File
+					RelativePath=".\include\Poco\Net\SocketProactor.h"
+					>
+				</File>
 			</Filter>
 			<Filter
 				Name="Source Files"
@@ -1265,6 +1269,10 @@
 				</File>
 				<File
 					RelativePath=".\src\SocketReactor.cpp"
+					>
+				</File>
+				<File
+					RelativePath=".\src\SocketProactor.cpp"
 					>
 				</File>
 			</Filter>

--- a/Net/testsuite/TestSuite_vs170.vcxproj
+++ b/Net/testsuite/TestSuite_vs170.vcxproj
@@ -942,6 +942,7 @@
     <ClInclude Include="src\ReactorTestSuite.h"/>
     <ClInclude Include="src\SMTPClientSessionTest.h"/>
     <ClInclude Include="src\SocketAddressTest.h"/>
+    <ClInclude Include="src\SocketProactorTest.h"/>
     <ClInclude Include="src\SocketReactorTest.h"/>
     <ClInclude Include="src\SocketsTestSuite.h"/>
     <ClInclude Include="src\SocketStreamTest.h"/>
@@ -1113,6 +1114,9 @@
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
     </ClCompile>
     <ClCompile Include="src\SocketAddressTest.cpp">
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>
+    </ClCompile>
+    <ClCompile Include="src\SocketProactorTest.cpp">
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
     </ClCompile>
     <ClCompile Include="src\SocketReactorTest.cpp">

--- a/Net/testsuite/TestSuite_vs170.vcxproj.filters
+++ b/Net/testsuite/TestSuite_vs170.vcxproj.filters
@@ -309,6 +309,9 @@
     <ClInclude Include="src\SocketReactorTest.h">
       <Filter>Reactor\Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="src\SocketProactorTest.h">
+      <Filter>Reactor\Header Files</Filter>
+    </ClInclude>
     <ClInclude Include="src\MailMessageTest.h">
       <Filter>Mail\Header Files</Filter>
     </ClInclude>
@@ -504,6 +507,9 @@
       <Filter>Reactor\Source Files</Filter>
     </ClCompile>
     <ClCompile Include="src\SocketReactorTest.cpp">
+      <Filter>Reactor\Source Files</Filter>
+    </ClCompile>
+    <ClCompile Include="src\SocketProactorTest.cpp">
       <Filter>Reactor\Source Files</Filter>
     </ClCompile>
     <ClCompile Include="src\MailMessageTest.cpp">

--- a/Net/testsuite/TestSuite_vs90.vcproj
+++ b/Net/testsuite/TestSuite_vs90.vcproj
@@ -1054,6 +1054,10 @@
 					RelativePath=".\src\SocketReactorTest.h"
 					>
 				</File>
+				<File
+					RelativePath=".\src\SocketProactorTest.h"
+					>
+				</File>
 			</Filter>
 			<Filter
 				Name="Source Files"
@@ -1064,6 +1068,10 @@
 				</File>
 				<File
 					RelativePath=".\src\SocketReactorTest.cpp"
+					>
+				</File>
+				<File
+					RelativePath=".\src\SocketProactorTest.cpp"
 					>
 				</File>
 			</Filter>

